### PR TITLE
MINI-4534: Remove deprecated methods from SDK

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
This PR aims to remove any deprecated method for applying the breaking changes targeting to upcoming `v4.0.0` and any associated unused code 

## Links
- MINI-4534

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ## CHANGELOG
 
-### 3.9.1 (2022-01-20)
+### 4.0.0 (In progress)
+**SDK**
+- **Removed:** Deprecated `getUniqueId` from `MiniAppMessageBridge` and `getAccessToken` from `UserInfoBridgeDispatcher`
```
Comment (corycaywood) [inline CHANGELOG.md:5]: The wording here could be a little confusing. Also, I think we should be specific about the function signature so it doesn't look like we removed the entire feature. How about this instead?
```suggestion
- **Removed:** Functions which were deprecated in previous releases have been removed:
  - `MiniAppMessageBridge.getUniqueId()`
  - `UserInfoBridgeDispatcher.getAccessToken(miniAppId: String, onSuccess: (tokenData: TokenData) -> Unit, onError: (message: String) -> Unit)`
  - `UserInfoBridgeDispatcher.getAccessToken(miniAppId: String, accessTokenScope: AccessTokenScope, onSuccess: (tokenData: TokenData) -> Unit, onError: (message: String) -> Unit)`
```

**Review by corycaywood — APPROVED** [review]
Looks good 👍  Just a suggestion on the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (khairul-alam-licon) [reply CHANGELOG.md]: Thank you, I have updated this part.

**Review by munir-rakuten — APPROVED** [review]
LGTM 👍

**Pull request merged**